### ISSUE TITLE
Customize Permission

### ISSUE THREAD

**Issue by zachsai** [domino54/slight-depth#1]
Hi, 
Can I have your permission to customize your resource pack and add support for modded blocks and ores?
Thanks!

**Comment by domino54** [comment]
Yes, you can customize the resource pack to support modded items!

However what I recommend instead is creating an extension resource pack that is not a direct fork, but references files from slight-depth to provide additional content without duplicating assets and having to work about keeping it up to date with the original. Drawback is that user will require both to work, but the benefit is easier maintenance and smaller size.
